integrations/node-fetch/require.ts

```diff
@@ -37,6 +38,9 @@ export function wrappedNodeFetch(fetch: any) {
     url: any,
     options: any
   ) {
+    if (process.env.KEPLOY_MODE == MODE_OFF) {
```

Comment:
Hey @re-Tick

It makes sense not to create a context here and directly calling the fetch function. I've incorporated this change, but a bit differently.

Reply:
If the user have not provided the KEPLOY_MODE env variable then, it will still do the same problem.

Reply:
That is correct. but probably that is the ideal behavior right? If the user isn't providing the KEPLOY_MODE then it's better to return some error and inform the user, instead of directly assuming OFF mode.

Reply:
Yes we should console an error to inform the user but also we should do the actual function call so, that the user application do not break due to Keploy SDK integration

Reply:
> That is correct. but probably that is the ideal behavior right? If the user isn't providing the KEPLOY_MODE then it's better to return some error and inform the user, instead of directly assuming OFF mode.

The default value of KEPLOY_MODE is "off". So, if the env variable is empty then, user application should run on off mode.